java-client/src/main/java/org/scassandra/http/client/PrimingRequest.java

```diff
@@ -281,6 +286,53 @@ public Result getResult() {
         public long getFixedDelay() {
             return fixedDelay;
         }
+
+        public static class ThenBuilder {
```

Comment:
It is weird that BatchPrimingRequest uses this but PrimingRequest doesn't, but understand it for API compatibility and I like the direction towards ThenBuilder.